__tests__/server/utils/getModulesToUpdate.spec.js

```diff
@@ -152,7 +152,7 @@ describe('getModulesToUpdate', () => {
         },
       },
     };
-    expect(getModulesToUpdate(current, next)).toEqual(['first-module', 'another-module', 'test-root']);
+    expect(getModulesToUpdate(current, next)).toEqual(['test-root']);
```

Comment:
Why did this expect change if the test case didn't change?

Reply:
good catch, looks like the mock outdated

Reply:
resolved 5072f48bf0080408e9584a2277514d411e3298ef